doc/python_reference.rst

```diff
@@ -29,6 +29,7 @@ Classes
    io.RawFIF
    io.RawArray
    Annotations
+   ElektaAverager
```

Comment:
You need to import this in `mne/__init__.py`. It currently can't be accessed there, and does not render properly:

https://2485-1301584-gh.circle-artifacts.com/0//home/ubuntu/mne-python/doc/_build/html/python_reference.html